README.md

```diff
@@ -12,15 +12,15 @@ Hound is an extremely fast source code search engine. The core is based on this
 
 ### Using Go Tools
 
-1. Use the Go tools to install Hound. The binaries `houndd` (server) and `hound` (cli) will be installed in your $GOPATH.
+1. Use the Go tools to install Hound. The binaries `houndd` (server) and `hound` (cli) will be installed in your $GOPATH/bin directory, which should be in your $PATH (`echo $PATH` to check).
 
 ```
 go get github.com/hound-search/hound/cmds/...
 ```
 
 2. Create a [config.json](config-example.json) in a directory with your list of repositories.
 
-3. Run the Hound server with `houndd` and you should see output similar to:
+3. Run the Hound server with `$GOPATH/bin/houndd` and you should see output similar to:
```

Comment:
Nit: can you change this so that it's not a run-on sentence?
> Run the Hound server with `$GOPATH/bin/houndd`. You should see output similar to:

Reply:
:D yes